YellowNotes.Core/Services/UserService.cs

```diff
@@ -33,5 +43,34 @@ public async Task<bool> ChangePassword(UserDto user, CancellationToken cancellat
         {
             return await repository.ChangePassword(user, cancellationToken);
         }
+
+        public string GenerateJWT(UserDto user)
+        {
+            var tokenHandler = new JwtSecurityTokenHandler();
+            var key = Encoding.ASCII.GetBytes(configuration["Jwt:SecretKey"]);
+            var tokenDescriptor = new SecurityTokenDescriptor
+            {
+                Subject = new ClaimsIdentity(new Claim[]
+                {
+                    new Claim(ClaimTypes.Email, user.Email)
+                }),
+                Expires = DateTime.UtcNow.AddHours(3),
+                SigningCredentials = new SigningCredentials(new SymmetricSecurityKey(key), SecurityAlgorithms.HmacSha256Signature)
+            };
+            var token = tokenHandler.CreateToken(tokenDescriptor);
+
+            return tokenHandler.WriteToken(token);
+        }
+
+        public bool ValidateToken(string token, UserDto user)
+        {
+            JwtSecurityToken securityToken = new JwtSecurityTokenHandler().ReadToken(token) as JwtSecurityToken;
```

Comment:
You don't have to explicitly declare every type. Use `var` to be consistent with your previous code. Look for other variables like this, cause there are many.